l10n_ro_stock_account_date/models/stock_picking.py

```diff
@@ -11,26 +14,39 @@ class StockPicking(models.Model):
     accounting_date = fields.Datetime(
         "Accounting Date",
         copy=False,
-        help="If this field is set, the svl and accounting entiries will "
+        help="If this field is set, the svl and accounting entries will "
         "have this date, If not will have the today date as it should be",
         tracking=True,
     )
 
+    def check_accounting_date(self, accounting_date=None):
+        date = accounting_date or self.accounting_date.date()
+        # if date > fields.date.today():
```

Comment:
Eu vreau sa pastrez asta, nu mi se pare ceva ok, sa poti pune datele din stock in viitor

Reply:
ok